src/db/sysdb_search.c

```diff
@@ -364,7 +364,7 @@ errno_t sysdb_getpwuid_with_views(TALLOC_CTX *mem_ctx,
                                   uid_t uid,
                                   struct ldb_result **res)
 {
-    int ret;
+    int ret = 0;
```

Comment:
Why have you decided to set ret to zero here?

Reply:
At end of function, inside done label, If we not setting any value of ret, It would throw uninitialized warning.

Reply:
So, please, set the ret to EOK in the appropriate place (just before calling "goto done")